libsubmit/providers/condor/condor.py

```diff
@@ -315,7 +252,7 @@ def submit(self, command, blocksize, job_name="parsl.auto"):
         channel_script_path = self.channel.push_file(script_path, self.channel.script_dir)
 
         cmd = "condor_submit {0}".format(channel_script_path)
-        retcode, stdout, stderr = self.channel.execute_wait(cmd, 3, envs=self.config['execution']['environment'])
+        retcode, stdout, stderr = self.channel.execute_wait(cmd, 3)
```

Comment:
I think we might need these envs set.

Reply:
Looking at this again, I see that there are two sets of envs: what is set in the constructor are envs to be set for the task, these are set before running commands in the channel. I think it makes more sense to set these in the channel, by adding an `envs` keyword arg to the channel (which brings the interface for `SSHChannel`, `SSHInteractiveLoginChannel`, and `LocalChannel` into sync). I've done this in f877068. Let me know if you have objections and I can change it back, or add a second, channel-specific `envs` keyword argument to the `Condor` provider.

Reply:
Got it.